### ISSUE TITLE
Migration and seed when working with mongodb

### ISSUE THREAD

**Issue by RafrafiDev** [w3tecch/express-typescript-boilerplate#51]
Hello all,

Please i would like to have some snipets of code that can help me to make migration as well as seeding some user data when working with mongodb.

I commented the call for migration because it gives me some errors when using migration with mongodb
```
 setup: {
            db: {
                script: series(
                    // 'nps db.migrate',
                    'nps db.seed'
                ),
                description: 'Setup`s the database by migrating and seeding'
            }
        },
```

and  changed the .env file:
```
DB_TYPE="mongodb"
DB_HOST="localhost"
DB_PORT=27017
DB_DATABASE="mydb"

```
and changed the seed code like this:
```
export class CreateBruce implements SeedsInterface {

    public async seed(factory: FactoryInterface): Promise<any> {
        const em = await factory.getConnection().mongoManager;

        const user = new User();
        user.firstName = 'Bruce';
        user.lastName = 'Wayne';
        user.email = 'bruce.wayne@wayne-enterprises.com';
        return await em.save(user);
    }

}

```

but when i run :
npm run setup
i get : 
```
executing seed:   CreateBruce
Could not run seed TypeError: this.databaseConnection.collection is not a function
The script called "db.seed" which runs "nps banner.seed && nps config && ts-node -F ./src/lib/seeds/cli.ts" failed with exit code 1 https://github.com/kentcdodds/nps/blob/v5.7.1/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
The script called "setup.db" which runs "nps db.seed" failed with exit code 1 https://github.com/kentcdodds/nps/blob/v5.7.1/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stack-nodejs-fondative@1.0.0 start: `nps "setup.db"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the stack-nodejs-fondative@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

**Comment by dweber019** [comment]
Closing as it's an issue in https://github.com/typeorm/typeorm/issues/1424

**Comment by cloakedch** [comment]
Both issues (this and typeorm/typeorm#1424) are not closed, yet the problem persists. Was that intended?

**Comment by hirsch88** [comment]
Hi @cloakedch 
We had a issue with the setup task, but this should be fixed in the newest release 3.0.0-rc.2.
Please have a look at let us know.
Thxs
